### DISCUSSION TITLE
Adding TXT record to Domain in Hobby account

### DISCUSSION THREAD

**Discussion by mycalldev** [vercel/next.js#65800] [Help]
### Summary

I have a domain that has valid configuration and in production. I wish to add a TXT record although when navigating to the > Settings > Domain > and clicking on the domain > Edit, I see no option to upload the TXT record. I'm sure in the past I have configured A records CNAME and additional records via this route.

**Comment by haydenbleasel** [comment]
Hey @mycalldev, DNS lives on the root-level of Vercel, not in the Project route.

So instead of "Home > {your project} > Settings > Domains", it's under "Home > Domains"

You can find it on this path: `https://vercel.com/{username}/~/domains`

![CleanShot 2024-05-16 at 17 21 13@2x](https://github.com/vercel/next.js/assets/4142719/75170257-9339-415a-a91b-f3cf867c3f98)

**Reply by MaydayTM** [reply]
great
